### DISCUSSION TITLE
Backup mail accounts/ filters? In sqllite?

### DISCUSSION THREAD

**Discussion by beardedfool** [jonaswinkler/paperless-ng#1130] [Support]
Just did the first ansible test install (sql) and want to confirm a couple of things to test migration to a new server/ reinstall please

Backups - readthedocs using export/ import. 
paperless.conf - all good here as defined in ansible install

sqlitedb - I can see this has most of the OCR content, 
I'm guessing mail accounts/ filters are stored here? So a backup/ restore should work there?

Are there other settings hidden elsewhere to be aware of please?

**Comment by jonaswinkler** [comment]
The SQL database has:

* All your document metadata
* All mail accounts and rules
* All tags / types / correspondents
* All saved views ("filters")
* No actual documents. These are stored in the /media folder, along with generated thumbnails.

If you move
* sql database
* media folder
* configuration file
* data folder (not necessarily required, everything except for the sqlite database, if used, can be regenerated).

to a new machine with the same version of paperless, you're all good.

**Comment by beardedfool** [comment]
Perfect, thanks for coming back so quickly and for all you've done, it's
amazing.

Whilst you're there, and happy to but this in another ticket if you want...

A lot of statements will have
Start date TO end date.

Any easy way to train it to use the second date, after 'TO'

Or is there any way to ask it to pick up a date from a specific XY
coordinate on the scan?
Maybe another tool to scrape that data/ recognise logos?

Thanks again.
